app/channels/deploy_notifications_channel.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 class DeployNotificationsChannel < ActionCable::Channel::Base
   def self.broadcast(count)
-    ActionCable.server.broadcast channel_name, count: count
+    ActionCable.server.broadcast(channel_name, {count: count})
```

Comment:
The `message` sent via the the [ActionCable::Server::Broadcasting#broadcast](https://api.rubyonrails.org/classes/ActionCable/Server/Broadcasting.html#method-i-broadcast) method is a Hash.

Explicitly send a hash in this method call.